README.md

```diff
@@ -161,13 +213,75 @@ jooq {
 }
 ```
 
+Kotlin DSL
+```kotlin
+import nu.struder.gradle.jooq.*
```

Comment:
should be `studer` ?

Reply:
Nice catch, thanks!

Reply:
Readme is fixed but the actual package name is still misspelt.

Reply:
Ah, yeah sorry fixed now